src/evergreen_mcp/server.py

```diff
@@ -28,6 +28,63 @@
 DEFAULT_PROJECT_ID = None
 
 
+def detect_project_from_workspace(
```

Comment:
Let's remove this method. Detection of project from workspace is role of agent and not MCP